node/cmd/guardiand/node.go

```diff
@@ -1003,6 +1011,40 @@ func runNode(cmd *cobra.Command, args []string) {
 		}
 	}
 
+	var gatewayRelayerWormchainConn *wormconn.ClientConn
+	if *gatewayRelayerContract != "" {
+		if *wormchainURL == "" {
```

Comment:
Do we want to add any of this to the rpcMap[] for the dumpRpcs command?

Reply:
Well, we don't do it for the accountant or wormchain in general, so doesn't seem like we need it for this?